app/models/effective/datatables/people_data_table.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+module Effective
+  module Datatables
+    # datatable for displaying all people
+    class PeopleDataTable < Effective::MongoidDatatable
+      include ApplicationHelper
+      include HtmlScrubberUtil
+      include DropdownHelper
+
+      datatable do
+        table_column :name, :label => l10n('hbx_profiles.people.table.name'), :proc => proc { |row| row.full_name }, :filter => false, :sortable => false
+        table_column :dob, :label => l10n('hbx_profiles.people.table.dob'), :proc => proc { |row| row.dob }, :filter => false, :sortable => false
+        table_column :hbx_id, :label => l10n('hbx_profiles.people.table.hbx_id'), :proc => proc { |row| row.hbx_id }, :filter => false, :sortable => false
+        table_column :role_type, :label => l10n('hbx_profiles.people.active_roles'), :proc => proc { |row| all_roles(row) }, :filter => false, :sortable => false
+        table_column :actions, :label => l10n('actions'), :width => '50px', :proc => proc { |row|
+          dropdown = [
+            [sanitize_html("<div class='#{pundit_class(Family, :can_update_ssn?)}'> Edit DOB / SSN </div>"), edit_dob_ssn_path(id: row.id, row_actions_id: "person_actions_#{row.id}"), can_display_edit_dob_ssn?(row)]
+          ]
+
+          render partial: 'datatables/shared/dropdown', locals: {dropdowns: @bs4 ? map_legacy_dropdown(dropdown) : dropdown, row_actions_id: "person_actions_#{row.id}"}, formats: :html
```

Comment:
@saipraveen18 We do not need `non-bs4` code anymore. We still have the `non-bs4` code in our codebase because the cucumbers depend on the `non-bs4` code. Please clean up this to not depend on bs4.